api/src/pcapi/core/users/external/zendesk_sell.py

```diff
@@ -0,0 +1,576 @@
+    INTERNAL_COMMENT = "Commentaire interne"
+    JURIDIC_NAME = "Nom juridique"
+    PC_PRO_STATUS = "Statut PC Pro"
+    PRODUCT_OFFERER_ID = "Produit Offerer ID"
+    PRODUCT_VENUE_ID = "Produit Venue ID"
+    REGION = "Région"
+    SIREN = "SIREN"
+    SIRET = "SIRET"
+    TYPAGE = "Typage"
+    BACKOFFICE_LINK = "Lien vers page BO"
+
+
+def configure_session() -> requests.Session:
+    session = requests.Session()
+    session.headers.update(
+        {
+            "Authorization": "Bearer " + ZENDESK_SELL_API_KEY,
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+        }
+    )
+
+    return session
+
+
+def build_url(path: str) -> str:
+    return urllib.parse.urljoin(ZENDESK_SELL_API_URL, path)
+
+
+def query_api(
+    method: str, path: str, body: str | dict | None
+) -> tuple[int, CaseInsensitiveDict[str] | Any, bytes | Any]:
+
+    session = configure_session()
+    try:
+        match method.upper():
+            case "PUT":
+                response = session.put(build_url(path), json=body)
+            case "POST":
+                response = session.post(build_url(path), json=body)
+            case "GET":
+                response = session.get(build_url(path))
+            case _:
+                raise ValueError("Unsupported method")
+        if not response.ok:
+            logger.error(
+                f"There was a {response.status_code} error in requesting the Zendesk Sell API",
+                extra={
+                    "status_code": response.status_code,
+                    "response": response.content,
+                    "body": json.dumps(body, indent=4),
+                },
+            )
+        response.raise_for_status()
+    except IOError as e:
+        raise e
+    if "json" in response.headers.get("Content-Type", "") and response.status_code != 204:
+        resp_body = response.json()
+    else:
+        resp_body = response.content
+
+    return response.status_code, response.headers, resp_body
+
+
+def set_venue_custom_fields(venue: offerers_models.Venue, created: bool = False) -> dict:
+    params = {
+        ZendeskCustomFieldsNames.DEPARTEMENT.value: venue.departementCode,
+        ZendeskCustomFieldsNames.INTERNAL_COMMENT.value: "Mis à jour par le produit le %s"
+        % (datetime.date.today().strftime("%d/%m/%Y")),
+        ZendeskCustomFieldsNames.HAS_PUBLISHED_COLLECTIVE_OFFERS.value: len(venue.collectiveOffers) > 0,
+        ZendeskCustomFieldsNames.JURIDIC_NAME.value: venue.name,
+        ZendeskCustomFieldsNames.PC_PRO_STATUS.value: get_venue_status(venue),
+        ZendeskCustomFieldsNames.PRODUCT_VENUE_ID.value: venue.id,
+        ZendeskCustomFieldsNames.SIRET.value: venue.siret,
+        ZendeskCustomFieldsNames.REGION.value: get_region_name_from_departement(venue.departementCode),
+        ZendeskCustomFieldsNames.TYPAGE.value: ["Lieu"],
+        ZendeskCustomFieldsNames.BACKOFFICE_LINK.value: f"{BACKOFFICE_URL}venues/{venue.id}",
```

Comment:
Je suggère d'utiliser `urllib.parse.urljoin()` : 

```suggestion
        ZendeskCustomFieldsNames.BACKOFFICE_LINK.value: urllib.parse.urljoin(BACKOFFICE_URL, f"/venues/{venue.id}"),
```

Car si jamais quelqu'un modifie BACKOFFICE_URL comme suit : 

```diff
- BACKOFFICE_URL=https://bo.example.com/
+ BACKOFFICE_URL=https://bo.example.com
```

... l'URL construite ici ne fonctionnera plus car il manquera le slash.